config/app.php

```diff
@@ -14,7 +14,7 @@
     */
 
     // 'name' => env('APP_NAME', 'Agri Poll'),
-    'name' => 'Agri Poll',
+    'name' => 'Buuza Agripoll',
```

Comment:
You hard coded it here

Reply:
The reason is that it is appearing on all the title pages and the App name is not used anywhere in the project FYI.